docs/admin/admin-ui-setup/credentials-setup.md

```diff
@@ -2,34 +2,56 @@
 
 Credentials are used to unlock protected resources such as applications, containers, and other assets.
 
-## Types of credentials
-
-The Credential manager in the Run:ai environment supports 4 types of credentials:
+The *Credentials* manager in the Run:ai environment supports 4 types of credentials:
 
 1. Docker registry.
 2. Access key.
 3. User name and password.
-4. [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret){target=_blank}.
+<!-- 4. [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret){target=_blank}.
+-->
 
-## Configuring Credentials
+## Secrets
+
+Credentials are built using `Secrets`. A `Secret` is an object that contains a small amount of sensitive data so that you don't need to include confidential data in your application code. When creating a credential you can either **create a new secret** or use an [existing secret](#existing-secrets).
+
+### Existing secrets
+
+An existing secret is a secret that you have created before creating the credential. One way to create a secret is to use the [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/#working-with-secrets){target=_blank}
+creation tool to create a pre-existing secret for the credential. You must `label` these secrets so that they are registered in the Run:ai environment.
+
+The following command makes the secret available to all projects in the cluster.
+
+```console
+kubectl label secret -n runai <SECRET_NAME> run.ai/cluster-wide-credentials=true
+```
```

Comment:
```suggestion

The following command makes the secret available to all projects in a department.
run.ai/resource: <credential type> (the options are password / access-key / docker-image)
run.ai/department: <department-id>
```

Reply:
This will enable the automatic replication of this secret to all the relevant subsidiaries.